docs/999-big-list-of-options.md

```diff
@@ -1467,12 +1467,16 @@ console.log(foo);
 Note that despite the name, this option does not "add" side effects to modules that do not have side effects. If it is important that e.g. an empty module is "included" in the bundle because you need this for dependency tracking, the plugin interface allows you to designate modules as being excluded from tree-shaking via the [`resolveId`](guide/en/#resolveid), [`load`](guide/en/#load) or [`transform`](guide/en/#transform) hook.
 
 **treeshake.propertyReadSideEffects**<br>
-Type: `boolean`<br>
+Type: `boolean | 'always'`<br>
 CLI: `--treeshake.propertyReadSideEffects`/`--no-treeshake.propertyReadSideEffects`<br>
 Default: `true`
 
+If `true`, only retain property reads that are determined to be needed. Note: this setting assumes that all getters are side effect free.
```

Comment:
This is somewhat confusing and also not true, we do not assume all getters are side-effect free. `true` means retain property reads that Rollup can resolve to have side-effects.

```suggestion
If `true`, retain unused property reads that Rollup can determine to have side-effects. This includes accessing properties of `null` or `undefined` or triggering explicit getters via property access. Note that this does not cover destructuring assignment or getters on objects passed as function parameters.
```

Reply:
The wording was changed as requested. I struggled to describe this option because of the known getter issues.